dashboards/aws-lex/aws-lex.json

```diff
@@ -0,0 +1,297 @@
+{
+  "name": "Amazon Lex",
+  "description": null,
+  "pages": [
+    {
+      "name": "Amazon Lex",
+      "description": null,
+      "widgets": [
+        {
+          "title": "Number Of Times Channel Used",
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "width": 4,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.pie"
+          },
+          "rawConfiguration": {
+            "colors": {
+              "seriesOverrides": [
+                {
+                  "color": "#6cff00",
+                  "seriesName": "FACEBOOK"
+                }
+              ]
+            },
+            "facet": {
+              "showOtherSeries": true
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [],
+                "query": "FROM Metric select count(aws.lex.Source) as `Channels Used For Conversation` facet aws.lex.Source"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "Number Of Request On Channel",
+          "layout": {
+            "column": 5,
+            "row": 1,
+            "width": 4,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.stacked-bar"
+          },
+          "rawConfiguration": {
+            "colors": {
+              "seriesOverrides": [
+                {
+                  "color": "#03ff17",
+                  "seriesName": "FACEBOOK"
+                }
+              ]
+            },
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [],
+                "query": "FROM Metric SELECT count(aws.lex.BotChannelRequestCount) as BotChannelRequestCount facet  aws.lex.Source, aws.lex.BotAlias TIMESERIES"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "",
+          "layout": {
+            "column": 9,
+            "row": 1,
+            "width": 4,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "rawConfiguration": {
```

Comment:
Could you move this markdown widget to be on the left hand side, so it's the first thing that people see when scanning the dashboard? It should be the top left corner.

Reply:
following this standard so far for all the entities